mayastor/src/grpc/mayastor_grpc.rs

```diff
@@ -149,9 +150,9 @@ impl mayastor_server::Mayastor for MayastorSvc {
         let args = request.into_inner();
         let rx = rpc_submit::<_, _, LvsError>(async move {
             if let Some(pool) = Lvs::lookup(&args.name) {
-                let _e = pool.destroy().await?;
+                pool.destroy().await?;
```

Comment:
First remove the pool from the config, please. If we fail to destroy the pool and crash halfway through, we at least are sure the pool is not loaded upon restart.

Reply:
Done